src/main/java/org/apache/sling/feature/cpconverter/accesscontrol/DefaultAclManager.java

```diff
@@ -166,32 +194,46 @@ private void addStatements(@NotNull SystemUser systemUser,
             return;
         }
 
-        Map<AccessControlEntry, String> entries = new LinkedHashMap<>();
+        Map<AccessControlEntry, String> resourceEntries = new LinkedHashMap<>();
+        Map<AccessControlEntry, String> principalEntries = new LinkedHashMap<>();
+
         authorizations.forEach(entry -> {
             String path = getRepoInitPath(entry.getRepositoryPath(), systemUser);
             if (path != null) {
```

Comment:
can/will `getRepoInitPath` ever return null? maybe it has the wrong nullability annotation?